apps/namadillo/src/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { init as initShared } from "@namada/shared/src/init-inline";
+import { SdkProvider } from "hooks/useSdk";
+import React from "react";
+import { createRoot } from "react-dom/client";
+import { RouterProvider } from "react-router-dom";
+import { getRouter } from "./App/AppRoutes";
+import reportWebVitals from "./reportWebVitals";
+import { ExtensionEventsProvider, IntegrationsProvider } from "./services";
+
+import "@namada/components/src/base.css";
+import "./tailwind.css";
+
+// TODO: we could show the loading screen while initShared is pending
+const container = document.getElementById("root");
+if (container) {
+  const root = createRoot(container);
+  initShared().then(() => {
```

Comment:
I _think_ we can remove this as init is being invoked by the SDK, but that may have unintended consequences as `Query` is still being used in places. We can leave it until those queries are updated!

Reply:
#777